flixel/FlxCamera.hx

```diff
@@ -653,6 +665,8 @@ class FlxCamera extends FlxBasic
 	{
 		super();
 		
+		pixelPerfectRender = (FlxG.renderBlit);
```

Comment:
I don't understand, should I just not initialize it? 

I did a search and FlxCamera.pixelPerfectRender is not set anywhere that I can find.

In the previous code pixelPerfectRender is initialized based on the compiler constant, but if we replace those with runtime constants, in order to maintain the same logic I have to move this to the constructor. Otherwise, we'll get different behavior as it's not set anywhere else.

Am I missing something?